hosted-ledgers/client-for-hosted-ledger.js

```diff
@@ -1,5 +1,8 @@
 const IlpPacket = require('ilp-packet')
-const plugin = require('./plugins.js').xrp.Customer()
+const HostedLedgerPlugin = require('ilp-plugin-payment-channel-framework')
+const plugin = new HostedLedgerPlugin({
+  server: 'btp+ws://:@localhost:9000/'
+})
```

Comment:
Why don't you add this to the plugins.js file rather?

`const plugin = require('./plugins.js').hosted.Customer()`

Reply:
Because at this point, the plugins file has already been edited by the user, and also there is nothing per-user about this; for xrp, every tutorial user has different testnet credentials, but here, everybody will just set it to localhost:9000.